egs/wsj/s5/local/wsj_prepare_dict.sh

```diff
@@ -81,6 +81,13 @@ grep -v ';;;' $dir/cmudict/cmudict.0.7a | \
 (echo '!SIL SIL'; echo '<SPOKEN_NOISE> SPN'; echo '<UNK> SPN'; echo '<NOISE> NSN'; ) | \
  cat - $dir/lexicon1_raw_nosil.txt | sort | uniq > $dir/lexicon2_raw.txt || exit 1;
 
+# Axel Horndasch (axel.horndasch@fau.de):
+# This is probably the worst hack ever made; but it will help me test word classes
+# for Kaldi on the WSJ corpus; please forgive me, but some compounds from the word
+# class "US_STATE" like "WEST-VIRGINIA" were not in the lexicon ...
+cat $dir/lexicon2_raw.txt local/wclass/wsj_classes_lexicon.txt | sort | uniq > $dir/lexicon2_raw_tmp.txt
+mv $dir/lexicon2_raw_tmp.txt $dir/lexicon2_raw.txt
```

Comment:
We may revisit this after you rework other aspects of the scripts.  What I'd like is that rather than just getting it working, you come up with some scripts that are easy to understand and use, where the aspects relating to words classes are as cleanly as possible separated from other aspects (or, if mixed in, are are mixed in in as natural a way as possible).